test/e2e/app/app.go

```diff
@@ -17,7 +17,10 @@ import (
 	"github.com/cometbft/cometbft/abci/example/kvstore"
 	abci "github.com/cometbft/cometbft/abci/types"
 	"github.com/cometbft/cometbft/crypto"
+	cryptoenc "github.com/cometbft/cometbft/crypto/encoding"
 	"github.com/cometbft/cometbft/libs/log"
+	"github.com/cometbft/cometbft/libs/protoio"
+	cmtproto "github.com/cometbft/cometbft/proto/tendermint/types"
```

Comment:
Doesn't really matter very much right now, I think.

With the reorg that @mzabaluev will be doing, this will end up changing anyways and will be more consistent across all code that imports our generated Protobuf code.

Reply:
Actually, I just did a quick search on the SDK repo and it's used in production code, even some protobufs import them (e.g., _query_, _block_, and _evidence_ proto definitions).
So we should be good on e2e app.